staging/src/k8s.io/cli-runtime/go.mod

```diff
@@ -19,13 +19,15 @@ require (
 	k8s.io/apimachinery v0.0.0
 	k8s.io/client-go v0.0.0
 	k8s.io/kube-openapi v0.0.0-20220328201542-3ee0da9b0b42
+	k8s.io/kubectl v0.0.0
```

Comment:
as an aside, this is adding a dependency cycle between k8s.io/kubectl <-> k8s.io/cli-runtime ... the code should be structured so that we don't have cycles in our modules

Reply:
Oops, good catch, I'm intending to come back to this PR next week, I'll look into this as well, thanks

Reply:
@liggitt I've just pushed a new version of this where I've moved the code needed from kubectl into cli-runtime and then put deprecated aliases in place of the old functions, does that seem like an appropriate way to solve the import cycle? That's now the first commit in the branch as a prefactor step